API/credentials.js

```diff
@@ -0,0 +1,4 @@
+window.hereCreds = {
```

Comment:
I think tyler mentioned this earlier but i dont think its good to keep your api keys hidden especially since we have the sire deployed now.